packages/components/src/image/src/Image.css

```diff
@@ -9,6 +9,6 @@
 }
 
 /* DARK MODE */
-.o-ui-dark-color-scheme .o-ui-image {
+.o-ui-shargate-dark .o-ui-image {
```

Comment:
2 issues here:

1- you misspelled "sharegate"
2- I don't like the fact that this selector is coupled to a specific theme. I would prefer a generic class which is not dependent of the theme name

Reply:
You would add a class to the image based on the colorscheme?

Reply:
It seems to be your intent.

Why did you made this change?

Reply:
`dark-color-scheme` was not applied due to an old renaming of things, my intent was to fix this regression, I will go the token route, as it's a more robust solution.

Reply:
This is one way to do it. you could also update the `getColorSchemeClassName` function used by the ThemeProvider to render an additional `o-ui-[light|dark]` class.

Since this class wouldn't be specific to a theme, it's fine to use it anywhere.